docs/pages/installation/eks.md

```diff
@@ -9,106 +9,77 @@ list: include
-    $ kubectl config set-context --current --user=meshery
-
-    o/p:
-    Context "aws" modified.
-    </div></div>
-    </pre>
-
-1. Generate kubeconfig yaml file to use as input to Meshery.
-    
-    <pre class="codeblock-pre">
-    <div class="codeblock"><div class="clipboardjs">
-    $ kubectl config view --minify --flatten >  config_aws_eks.yaml
-    </div></div>
-    </pre>
-
-Meshery should now be connected with your managed Kubernetes instance. Take a look at the [Meshery guides]({{ site.baseurl }}/guides) for advanced usage tips.-->
+{% include installation_prerequisites.html %}
 
-*Coming soon*
\ No newline at end of file
+## To set up and run Meshery on EKS:
+
+- Connect Meshery to your EKS cluster
+  - [Meshery CLI (mesheryctl)](#connect-meshery-to-azure-kubernetes-cluster)
+  - [eks CLI (eksctl)](https://eksctl.io/introduction/#installation)
+  - [AWS CLI (aws)](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html)
+- [Install Meshery on your EKS cluster](#install-meshery-into-your-eks-cluster)
+- [Access Meshery's UI](#port-forward-to-the-meshery-ui)
+
+### Connect Meshery to Elastic Kubernetes Cluster
+
+The following set of instructions expects you to have created a EKS cluster in your resource group Configure Meshery to connect to your EKS cluster by executing:
+
+ <pre class="codeblock-pre"><div class="codeblock">
+ <div class="clipboardjs">
+ mesheryctl system config eks
+ </div></div>
+ </pre>
+
+#### Manual Steps
+
+Alternatively, you may execute the following steps to manually configure Meshery to connect to your AKS cluster.
+
+- Install [AWS CLI(aws)](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html), and login
+  to your AWS account. Install [EKS CLI (eksctl)](https://eksctl.io/introduction/#installation) too to access eks cluster easily.
+
+- After successfull login, you have to select the subscription with which your AKS is associated with by configuring your AWS CLI to your AWS account. Refer [this link](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) for more
+
+- Get/Update the kubeconfig from your EKS cluster by connecting with AWS CLI
+<pre class="codeblock-pre"><div class="codeblock">
+<div class="clipboardjs">
+aws eks --region region update-kubeconfig --name cluster_name
+</div></div>
+</pre>
+
+- Set your cluster context and check your cluster-info
+<pre class="codeblock-pre"><div class="codeblock">
+<div class="clipboardjs">
+kubectl set-context EKS_SERVICE_NAME
+kubectl cluster-info
+</div></div>
+</pre>
+
+### Install Meshery into your AKS cluster
+
+ <pre class="codeblock-pre"><div class="codeblock">
+ <div class="clipboardjs">
+ $ kubectl create ns meshery
+ $ helm repo add meshery https://meshery.io/charts/
+ $ helm install meshery --namespace meshery
+ </div></div>
+ </pre>
+ - Meshery server supports customizing authentication flow callback URL, which can be configured in the following way
```

Comment:
Good. Customizing callback should be its own section and should not hidden in the collapsed manual steps.